app/metrics.yaml

```diff
@@ -1835,7 +1835,7 @@ search.default_engine:
       - erichards@mozilla.com
     expires: never
   submission_url:
-    type: string
+    type: url
```

Comment:
Unfortunately changing the type means it will require a new name (that's a shortcoming of our current pipeline)